_people/michael_longhurst.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Michael Longhurst

```

Comment:
You've got some weird control character here `^M`. Probably a windows thing. Needs removing.